src/components/transactions/TxSummary/index.tsx

```diff
@@ -42,7 +42,7 @@ const TxSummary = ({ item, isGrouped }: TxSummaryProps): ReactElement => {
       })}
       id={tx.id}
     >
-      {nonce && !isGrouped && (
+      {nonce != null && !isGrouped && (
```

Comment:
We really have to stop doing implicit boolean coercion with numbers and bignumber. This is now the 6th(?) bug I see around this.

I looked into it and found this eslint rule:

```
    "@typescript-eslint/strict-boolean-expressions": [
      "warn",
      {
        "allowString": true,
        "allowNumber": false,
        "allowNullableObject": true,
        "allowNullableBoolean": true,
        "allowNullableString": true,
        "allowNullableNumber": false,
        "allowNullableEnum": true
      }
    ],
```


It would have caught this error. But it requires [type checked linting](https://typescript-eslint.io/linting/typed-linting) so it is slower.

Reply:
I think this is more accurate as `nonce` is of type `number | undefined`
```suggestion
      {nonce !== undefined && !isGrouped && (
```